extensions/amp-story-auto-ads/0.1/amp-story-auto-ads-ad-badge.css

```diff
@@ -15,16 +15,19 @@
  */
 
 .i-amphtml-ad-overlay-container {
-  height: 24px !important;
+  align-items: center;
```

Comment:
Do you want to use !important for these rules as well?

Reply:
yep, added.